src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolJrdOfficeOrgRoleMappingTest.java

```diff
@@ -628,4 +681,539 @@ void shouldReturnRoles_withPresidentAppointment_withResidentRole_withIAC() {
             }
         });
     }
+
+    @Test
+    @DisplayName("Scenario 2: Missing Service code in Appointment")
+    void shouldCreateAssignmentsWhenAppointmentServiceCodeIsMissing() {
+
+        judicialAccessProfiles.forEach(judicialAccessProfile -> {
+            judicialAccessProfile.setAppointment("Employment Judge");
+            judicialAccessProfile.setAppointmentType("fee paid");
+            judicialAccessProfile.setServiceCode(null);
+            judicialAccessProfile.setAuthorisations(
+                    List.of(Authorisation.builder().userId(UUID.randomUUID().toString()).serviceCode("BFA1").build(),
```

Comment:
userId is not required and its planned to remove from Authorisation. So please remove .userId(UUID.randomUUID().toString())